example/docker-compose/distributed/docker-compose.yaml

```diff
@@ -101,3 +101,40 @@ services:
       - GF_AUTH_DISABLE_LOGIN_FORM=true
     ports:
       - "3000:3000"
+
+  etcd1:
```

Comment:
this could be reduced to a single node etcd cluster for simplicity. not a big deal either way.

Reply:
Changed docker-compose for extending the e2e tests to use all options for the kv store.